INSTALL.md

```diff
@@ -37,7 +37,7 @@ There is a rudimentary docker-compose configuration for production usage (`docke
 
 
 1. Configure OSEM
-   You have at least to set `SECRET_KEY_BASE`
+   You have at least to set `OSEM_DB_PASSWORD` and `SECRET_KEY_BASE`
```

Comment:
`OSEM_DB_PASSWORD` may or may not be needed depending on the DB adapter you are using

Reply:
Note the [context](https://github.com/openSUSE/osem/blob/38b648655060d06109734821095cf32e7a82efb6/INSTALL.md#deploy-via-dockerdocker-compose), though—a section about:

1. using the example Docker Compose configuration, which specifies PostgreSQL
1. production deployment, for which the fallback to a [hard-coded secret](https://github.com/openSUSE/osem/blob/38b648655060d06109734821095cf32e7a82efb6/config/database.yml#L15) is particularly unwise